packages/e2e-playwright/tests/dropin/sessions/dropin.sessions.spec.ts

```diff
@@ -0,0 +1,79 @@
+import { test, expect } from '../../../pages/dropin/dropin.fixture';
+import { REGULAR_TEST_CARD, TEST_CVC_VALUE, TEST_DATE_VALUE } from '../../utils/constants';
+import { typeIntoSecuredField } from '../../../models/dropinModelUtils/utils';
+
+import LANG from '../../../../lib/src/language/locales/en-US.json';
+
+const CARD_IFRAME_TITLE = LANG['creditCard.encryptedCardNumber.aria.iframeTitle'];
+const EXPIRY_DATE_IFRAME_TITLE = LANG['creditCard.encryptedExpiryDate.aria.iframeTitle'];
+const CVC_IFRAME_TITLE = LANG['creditCard.encryptedSecurityCode.aria.iframeTitle'];
+
+const CARD_IFRAME_LABEL = LANG['creditCard.cardNumber.label'];
+const EXPIRY_DATE_IFRAME_LABEL = LANG['creditCard.expiryDate.label'];
+const CVC_IFRAME_LABEL = LANG['creditCard.securityCode.label'];
+
+test.describe('Dropin Sessions flow', () => {
+    test('#1 Should succeed in making a payment', async ({ dropinSessions_regular }) => {
+        const { dropin, page } = dropinSessions_regular;
+
+        await dropin.isComponentVisible();
+
+        const creditCard = dropin.getPaymentMethodItem('Cards');
+        await creditCard.scrollIntoViewIfNeeded();
+
+        await page.waitForTimeout(500); // needs this else card number isn't guaranteed to fill correctly !?
```

Comment:
Do you know if this is an issue only with filling secured fields or also other input fields too?

Reply:
I don't know